src/main/java/uk/gov/pay/adminusers/service/ServiceUpdater.java

```diff
@@ -44,24 +44,25 @@
 
     @Inject
     public ServiceUpdater(ServiceDao serviceDao) {
-        ImmutableMap.Builder<String, BiConsumer<ServiceUpdateRequest, ServiceEntity>> attributeUpdaters = ImmutableMap.builder();
-        attributeUpdaters.put(FIELD_GATEWAY_ACCOUNT_IDS, assignGatewayAccounts());
-        attributeUpdaters.put(FIELD_CUSTOM_BRANDING, updateCustomBranding());
-        attributeUpdaters.put(FIELD_REDIRECT_NAME, updateRedirectImmediately());
-        attributeUpdaters.put(FIELD_COLLECT_BILLING_ADDRESS, updateCollectBillingAddress());
-        attributeUpdaters.put(FIELD_CURRENT_GO_LIVE_STAGE, updateCurrentGoLiveStage());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_NAME, updateMerchantDetailsName());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_ADDRESS_LINE_1, updateMerchantDetailsAddressLine1());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_ADDRESS_LINE_2, updateMerchantDetailsAddressLine2());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_ADDRESS_CITY, updateMerchantDetailsAddressCity());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_ADDRESS_COUNRTY, updateMerchantDetailsAddressCountry());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_ADDRESS_POSTCODE, updateMerchantDetailsAddressPostcode());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_EMAIL, updateMerchantDetailsEmail());
-        attributeUpdaters.put(FIELD_MERCHANT_DETAILS_TELEPHONE_NUMBER, updateMerchantDetailsPhone());
+        Map<String, BiConsumer<ServiceUpdateRequest, ServiceEntity>> attributeUpdaters = new HashMap<>() {{
+                put(FIELD_GATEWAY_ACCOUNT_IDS, assignGatewayAccounts());
+                put(FIELD_CUSTOM_BRANDING, updateCustomBranding());
+                put(FIELD_REDIRECT_NAME, updateRedirectImmediately());
+                put(FIELD_COLLECT_BILLING_ADDRESS, updateCollectBillingAddress());
+                put(FIELD_CURRENT_GO_LIVE_STAGE, updateCurrentGoLiveStage());
+                put(FIELD_MERCHANT_DETAILS_NAME, updateMerchantDetailsName());
+                put(FIELD_MERCHANT_DETAILS_ADDRESS_LINE_1, updateMerchantDetailsAddressLine1());
+                put(FIELD_MERCHANT_DETAILS_ADDRESS_LINE_2, updateMerchantDetailsAddressLine2());
+                put(FIELD_MERCHANT_DETAILS_ADDRESS_CITY, updateMerchantDetailsAddressCity());
+                put(FIELD_MERCHANT_DETAILS_ADDRESS_COUNRTY, updateMerchantDetailsAddressCountry());
+                put(FIELD_MERCHANT_DETAILS_ADDRESS_POSTCODE, updateMerchantDetailsAddressPostcode());
+                put(FIELD_MERCHANT_DETAILS_EMAIL, updateMerchantDetailsEmail());
+                put(FIELD_MERCHANT_DETAILS_TELEPHONE_NUMBER, updateMerchantDetailsPhone());
+            }};
```

Comment:
Another `Map.ofEntries(…)` opportunity.

Reply:
Unfortunately we add more things to the map later on :(